### DISCUSSION TITLE
Normal sorting of sequence numbers

### DISCUSSION THREAD

**Discussion by suprunchuk** [dbeaver/dbeaver#23237] [Q&A] [answered]
### Description

Hello. I can't find where to do normal sorting in the settings.
I want my numbers to be displayed in the usual order of 1, 2, 3, 4, instead of 1, 13, 14..., 2, 20,21..
![1](https://github.com/dbeaver/dbeaver/assets/57152612/2ef5da40-613c-4b10-8c5d-60b3ef47a83a)

### DBeaver Version

2024.0.1

### Steps to reproduce

Open a table with ID ordinal numbers

**Comment by yawuxi** [comment]
Hey! Just sort column by clicking triangle next to the column name and choose: "Order by [column name] ASC"

If you want to store it even after table was closed, just click that button - 
![image](https://github.com/dbeaver/dbeaver/assets/67982524/45ce1c94-fa1d-4581-a05b-81a0a721c4cb)
in the top right corner

**Comment by E1izabeth** [comment]
The column type is a varchar or string. Make this column to be of a numeric type and it will work.

**Comment by E1izabeth** [comment, marked as answer]
Another approach is to add virtual column of casted strings to integer this way and then sort by this virtual column. This column will not exist in the database, it exists only in DBeaver.

![image](https://github.com/dbeaver/dbeaver/assets/28875055/6c6e5e0e-32a9-444b-9033-b89b1e47f6f8)

![image](https://github.com/dbeaver/dbeaver/assets/28875055/06aa46d1-021b-4f23-a2fa-565041e72514)

**Comment by suprunchuk** [comment]
> The column type is a varchar or string. Make this column to be of a numeric type and it will work.

@E1izabeth Understood. My column type is not number. This can't be fixed. It was and will be so. Thanks for the answer.
